airbyte-integrations/connectors/source-amazon-mws/source_amazon_mws/schemas/GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL.json

```diff
@@ -0,0 +1,158 @@
+{
+  "name": "GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL",
+  "supported_sync_modes": ["full_refresh", "incremental"],
+  "source_defined_cursor": true,
+  "default_cursor_field": ["purchase-date"],
```

Comment:
this should be `updated-at-date`. From the [docs](http://docs.developer.amazonservices.com/en_US/orders-2013-09-01/Orders_ListOrders.html): 

```
A date used for selecting orders that were last updated after (or at) a specified time. An update is defined as any change in order status, including the creation of a new order. Includes updates made by Amazon and by the seller.
```